services/migration_service/requirements.txt

```diff
@@ -1,4 +1,3 @@
-aiohttp
-aiohttp_swagger
-psycopg2
```

Comment:
isn't psycopg2 still required?

Reply:
Good catch, this was dropped at some point in the UI branch and probably kept working due to all service dependencies being installed in `setup.py`. I'll reintroduce the dependency.